crowdfunding_compassion/models/crowdfunding_participant.py

```diff
@@ -39,6 +39,7 @@ class CrowdfundingParticipant(models.Model):
     instagram_url = fields.Char(string="Instagram link")
     personal_web_page_url = fields.Char(string="Personal web page")
     profile_photo = fields.Binary(related="partner_id.image")
+    profile_photo_url = fields.Binary(compute="_compute_profile_photo_url")
```

Comment:
This field should be of type `fields.Char`, since the _URL_ is stored as a string and not the binary image directly.

Reply:
Oops